pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -1306,13 +1371,23 @@ func (c *Cluster) getExpandPVCInitContainer(osdProps osdProperties, osdID string
 
 	*/
 	osdDataPath := activateOSDMountPath + osdID
+	osdSize, err := convertSizeToBytes(osdProps.pvcSize)
+	if err != nil {
+		logger.Errorf("failed to parse osd size. %v", err)
+		return v1.Container{}
+	}
 
 	return v1.Container{
 		Name:            expandOSDInitContainer,
 		Image:           c.spec.CephVersion.Image,
 		ImagePullPolicy: controller.GetContainerImagePullPolicy(c.spec.CephVersion.ImagePullPolicy),
 		Command: []string{
-			"ceph-bluestore-tool",
+			"/bin/bash",
+			"-x",
+			"-e",
+			"-m",
+			"-c",
```

Comment:
I see this is copying code from here but without comments to explain options. With short-form options, comments are really helpful for maintaining code over many years.

https://github.com/rook/rook/blob/c5d27467d6f09360602346c4d97637bf66b6d3a2/pkg/operator/ceph/controller/spec.go#L820-L823